components/org.wso2.carbon.identity.oauth/src/test/java/org/wso2/carbon/identity/openidconnect/DefaultIDTokenBuilderTest.java

```diff
@@ -157,14 +173,59 @@ public void setUp() throws Exception {
 
     }
 
-    @Test
-    public void testBuildIDToken() throws Exception {
+    @DataProvider(name = "TestBuildIDToken")
```

Comment:
Doesn't look like we are using any data providers here. If so, we can remove this annotation